custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -226,8 +225,12 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
             vm.goToRoute = function (route) {
                 $location.path(route);
             };
+            vm.getBackArrowLink = function () {
+                return navigationService.getPagePath('program_summary/' + vm.sectionSlug, $location.search());
+            };
```

Comment:
This will fail for anything that doesn't have `vm.sectionSlug` defined so will bring those in each time I port a new report to mobile (it's never called on web so should be fine).